src/main/java/frontend/src/MenuBar.js

```diff
@@ -2,22 +2,28 @@ import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
+// import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
-import IconButton from '@material-ui/core/IconButton';
-import MenuIcon from '@material-ui/icons/Menu';
+import Logo from "./Logo";
+
+// import { CardMedia } from '@material-ui/core';
+//
+// import logo from './logo.png';
 
 const useStyles = makeStyles((theme) => ({
     root: {
         flexGrow: 1,
     },
-    menuButton: {
-        marginRight: theme.spacing(2),
-    },
-    title: {
-        flexGrow: 1,
-        textAlign: "start",
-    },
+    // menuButton: {
```

Comment:
Happens in a few places - I hope it's just a temporary thing, but should probably be cleaned up.